<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorOptions</class>
 <widget class="QWidget" name="ColorOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>117</width>
    <height>62</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scene Color</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="currentBackgroundLabel_">
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="setBackgroundColorPushButton_">
     <property name="text">
      <string>Scene Color</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
